plugins/CoreHome/vue/src/MatomoDialog/MatomoDialog.vue

```diff
@@ -22,6 +22,11 @@ export default defineComponent({
       type: Boolean,
       required: true,
     },
+    options: {
+      type: Object,
+      required: false,
+      default: function _default() { return {}; },
```

Comment:
guess this could be simplified to
```suggestion
      default: () => ({}),
```

Reply:
I previously had it without the second round brackets and TS was complaining.. this is better, thanks.